egs2/fleurs/asr1/README.md

```diff
@@ -0,0 +1,32 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Thu Jun 16 03:14:31 EDT 2022`
+- python version: `3.8.6 (default, Dec 17 2020, 16:57:01)  [GCC 10.2.0]`
+- espnet version: `espnet 202205`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `45e8cb9173a072f85ee7a7ccbcae06af7c5c484a`
+  - Commit date: `Wed Jun 1 14:21:14 2022 +0900`
+
+## asr_train_asr_transformer_raw_en_us_bpe150_sp
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_en_us_bpe150_valid.loss.ave_asr_model_valid.acc.ave/dev_en_us|388|7935|5.8|84.3|10.0|7.6|101.8|100.0|
+|decode_asr_lm_lm_train_lm_en_us_bpe150_valid.loss.ave_asr_model_valid.acc.ave/test_en_us|647|14344|6.0|83.7|10.2|7.6|101.6|100.0|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_en_us_bpe150_valid.loss.ave_asr_model_valid.acc.ave/dev_en_us|388|47051|41.3|26.5|32.3|6.6|65.3|100.0|
```

Comment:
Sorry forgot to mention, the results are very preliminary. Have not really explored ASR training and model parameters. There is about 75 hours of English data in the corpus.

Reply:
I see, if it has 75 hours, the results seems not very satisfactory. But it indeed may happen with different data conditions and domains. My current suggestion is that 150 bpe seems to be small for an English model. You might want to increase it to have a try (though it could not be the fundamental issue for now.

Reply:
Ok, I will take a look into the bpe as well. Will make update when training is finished

Reply:
Can you paste the learning curve?
I think this training is just corrupted and we need to tune the optimization hyperparameters (just learning rate).